### DISCUSSION TITLE
post/nju-ssl-vpn-protocol/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#11] [Announcements]
# post/nju-ssl-vpn-protocol/

由于南大给的 SSL VPN 实在是过于难用加上个人的强迫症, 就心血来潮分析了一下它的协议并重新实现了一个.(?) 简单记录下相关的分析过程. 抓包分析0x00 Wireshark 抓包启动 VPN 并登录账号, 使用 Wireshark 抓取物理网卡上的流量. 观察到有 TLS 1.3 和 TLS 1.1 两种流量, 都是发往 VPN 地址的 443 端口的.  TLS 1.3 应该是套壳 E

https://blog.lyc8503.site/post/nju-ssl-vpn-protocol/

**Comment by aaaaaaamua** [comment]
大佬受教了
